com.bitwig.BitwigStudio.yaml

```diff
@@ -0,0 +1,77 @@
+---
+app-id: com.bitwig.BitwigStudio
+command: bitwig-studio
+runtime-version: "20.08"
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+rename-desktop-file: bitwig-studio.desktop
+rename-icon: bitwig-studio
+finish-args:
+  # TODO: we need MIDI, Audio, DRI, and realtime performances
+  - --device=all
+  - --nodevice=kvm
+
+  # pipewire
+  - --filesystem=xdg-run/pipewire-0
+
+  # Needed to talk with pulseaudio or pipewire
+  - --share=ipc
+
+  # Needed for updating the sound content, checking for online notifications and authentication
+  - --share=network
+
+  # Self explanatory
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
my take here is that anything that embed audio plugin fail with Wayland anyway.

Reply:
True! But we do use third party dialogs for opening files which might use wayland right?

Reply:
they will be fine with x11 only.

Reply:
Is there anything bad in having the wayland socket?

Reply:
Is there a --socket=pipewire?

Reply:
no. it's the "filesystem=xdg-run/pipewire-0"

Reply:
Should be dropped as per @hfiguiere's comments.